corehq/apps/users/models.py

```diff
@@ -1041,6 +1041,8 @@ class CouchUser(Document, DjangoUserMixin, IsMemberOfMixin, EulaMixin):
     login_attempts = IntegerProperty(default=0)
     attempt_date = DateProperty()
 
+    commtrack_supply_point = StringProperty()
```

Comment:
based on the discussion below, should this be on CouchUser or only on CommCareUser?